### ISSUE TITLE
Branding (Logo, Favicons, Welcome Page)

### ISSUE THREAD

**Issue by man-oi** [me-and-company/web-project-kit#7]
A welcome page with logo and favicon to explain the kit to users

**Comment by robinstickel** [comment]
I know this is a bigger change but I would like to rethink the naming of the project. I would prefer a more recognizable name. Even we use "Starterkit" most of the time when speaking about the project.

This would most likely change the repository name and slug as well. So what do you guys think @wumble and @chrSchroers?

**Comment by man-oi** [comment]
Yes, I absolutely support the idea. I find the post-it with "grace" a good approach 💃
